### PR TITLE
inserted open brackets

### PR DESCRIPTION
**Pull request by RAJU2529**
after reading this article , i inserted  open  brackets in between website address

### CODE REVIEW

**Comment by e0i** [comment]
@RAJU2529 Please change the target branch of the PR from `live` to `master` to continue with the review. Thank you.

**Comment by RAJU2529** [comment]
@e0i . Changed to master. Thanks

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

File: docset/winserver2012-ps/smbshare/Get-SmbSession.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Accept wildcard characters: False
 
 ### -CimSession
 Runs the cmdlet in a remote session or on a remote computer.
-Enter a computer name or a session object, such as the output of a New-CimSessionhttp://go.microsoft.com/fwlink/p/?LinkId=227967 or Get-CimSessionhttp://go.microsoft.com/fwlink/p/?LinkId=227966 cmdlet.
+Enter a computer name or a session object, such as the output of a New-CimSession(http://go.microsoft.com/fwlink/p/?LinkId=227967) or Get-CimSession(http://go.microsoft.com/fwlink/p/?LinkId=227966)cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/Get-SmbSession.md]:
```suggestion
Enter a computer name or a session object, such as the output of a [New-CimSession](http://go.microsoft.com/fwlink/p/?LinkId=227967) or [Get-CimSession](http://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet.
```

**Comment by e0i** [comment]
Closing due to incorrect URL. Thank you.

**Pull request closed without merging**